src/compute/src/lib.rs

```diff
@@ -80,8 +80,12 @@ pub struct ComputeNodeOpts {
     pub file_cache_dir: String,
 
     /// Endpoint of the connector node
-    #[clap(long, default_value = "127.0.0.1:60061")]
-    pub connector_source_endpoint: String,
+    #[clap(long, env = "CONNECTOR_SOURCE_ENDPOINT")]
```

Comment:
https://github.com/risingwavelabs/risingwave/pull/6481 is merged, which starts connector node and do e2e testing. After we made this change, we may also have to modify the CI to specify the source endpoint via env var.

Reply:
We can also add some e2e testing after this PR.

Reply:
> We can also add some e2e testing after this PR.

Maybe the next one. This one is large enough.